demosplan/DemosPlanCoreBundle/Addon/AddonManifestCollection.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace demosplan\DemosPlanCoreBundle\Addon;
+
+use demosplan\DemosPlanCoreBundle\Utilities\DemosPlanPath;
+use Symfony\Component\Yaml\Yaml;
+
+// TODO: find a better name for this!!!
```

Comment:
I think we can drop this TODO now, @AlexMenschoid ?

Reply:
done in [4b4cdd8](https://github.com/demos-europe/demosplan-core/pull/640/commits/4b4cdd8e956919fa555529ec615ce9b1c459f421)